### DISCUSSION TITLE
Proper way to use AddDump with microstrip simulation

### DISCUSSION THREAD

**Discussion by danielb-28** [thliebig/openEMS-Project#21] [General]
I've recently (less than a week) started learning openEMS and I aim to use it for microstrip simulations. I tried to use the AddDump method on the MSL notch filter example but I'm a little confused with it. I would like to see the E field over time along the MSL. What is the correct way to get the vtk file (I suppose this is what I need to see the results with paraview) from the MSL notch filter simulation? Where can I read more about this?

Thanks!

**Comment by thliebig** [comment]
I guess you are using Matlab/Octave? Try `help AddDump` for more information about how to set it up.
Or look at the [definition/file](https://github.com/thliebig/CSXCAD/blob/master/matlab/AddDump.m) directly.

I hope that helps,
Thorsten

**Reply by thliebig** [reply]
The [Rect Waveguide Tutorial](https://github.com/thliebig/openEMS/blob/master/matlab/Tutorials/Rect_Waveguide.m) for example uses a time-domain dump. But with "FileType" 1 (hdf5) to read and display directly in Octave. You can easily change that to dump as vtk as well.

**Reply by danielb-28** [reply]
Hi Thorsten! Sorry, I forgot to mention it. I'm using the python interface...

Thanks for your quick reply!

**Reply by thliebig** [reply]
Well it's the same for python. Look at the docu for the ["Dump-Property"](https://docs.openems.de/python/CSXCAD/CSProperties/CSPropDumpBox.html#CSXCAD.CSProperties.CSPropDumpBox)

And/or the [Rect Waveguide Tutorial](https://docs.openems.de/python/openEMS/Tutorials/Rect_Waveguide.html)

**Reply by danielb-28** [reply]
I was missing the AddBox() after the AddDump()...

Thanks, Thorsten!
